shell/utils/validators/formRules/index.ts

```diff
@@ -458,7 +458,7 @@ export default function(t: Translation, { key = 'Value' }: ValidationOptions): {
     return runValidators(val, [startHyphen('label'), endHyphen('label'), startDot('label'), endDot('label'), required]);
   };
 
-  const repo: Validator = (val) => val && !/((git|ssh|http(s)?)|(git@[\w\.]+))(\:(\/\/)?)([\w\.@\:/\-~]+)(\.git)?(\/)?/.test(val) ? t('validation.setting.repo') : undefined;
+  const repo: Validator = (val) => val && !/((git|ssh|oci|http(s)?)|(git@[\w\.]+))(\:(\/\/)?)([\w\.@\:/\-~]+)(\.git)?(\/)?/.test(val) ? t('validation.setting.repo') : undefined;
```

Comment:
Tan made a good point in slack, fleet doesn't support oci for their _git repo_'s (helm repos though do)